server/database/migrations/2017_06_21_080948_alter_requirements_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AlterRequirementsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('requirements', function (Blueprint $table) {
+            $table->boolean('locked');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        //
```

Comment:
@khai-tvt  Please add down function to reverse:
Schema::table('requirements', function (Blueprint $table) {
            $table->dropColumn('locked');
});